CLAUDE.md

```
@@ -0,0 +1,119 @@
+
+```bash
+# Run core library tests
+cargo test
+
+# Run extension tests (requires single thread)
+cargo test --all-features --lib extensions -- --test-threads=1
+
+# Run doc tests with all features
+cargo test --doc --all-features
+
+# Run a specific test module
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)